packages/ui/src/primitives/field.tsx

```diff
@@ -18,7 +18,7 @@ export const Root = React.forwardRef<HTMLDivElement, React.HTMLAttributes<HTMLDi
       className={cx(
         '[--field-input-height:1.875rem]',
         '[--field-input-px:theme(spacing.3)]',
-        '[--field-input-py:theme(spacing[1.5])]',
+        '[--field-input-py:theme(spacing.1)]',
```

Comment:
Maybe we only opt in to these spacing changes for ios instead of by default? Same with any of the other changes for sizing and spacing?